webpack.config.cjs

```diff
@@ -105,6 +105,15 @@ const sharedConfig = {
         use: ['source-map-loader'],
         enforce: 'pre',
       },
+      {
+        test: /\.worker\.js$/,
+        use: {
+          loader: 'worker-loader',
+          options: {
+            inline: 'fallback',
```

Comment:
This is required, as it convert worker a blob, to get around cross origin issues. [See](https://github.com/webpack-contrib/worker-loader#cross-origin-policy)